packages/react-storage/src/components/StorageBrowser/context/actions/listLocationsAction.ts

```diff
@@ -16,7 +16,7 @@ const PAGE_SIZE = 1000;
 export interface ListLocationsActionOptions<T>
   extends Omit<ListActionOptions<T>, 'delimiter'> {}
 
-export interface ListLocationsActionInput<T = never>
+export interface ListLocationsActionInput<T = Permission>
```

Comment:
[type fix]
In ListLocationsActionInput, exclude has type `never | never[]`

Reply:
Do we still need to remove providing the generic on line 30 with this change?

Reply:
Yeah this is required, keeping the generic in line 30 brings back the error 
`Type 'ListLocationsActionInput<Permission>' is not assignable to type 'ListLocationsActionInput<T>'.`